examples/image-resize/app/components/image.tsx

```diff
@@ -1,26 +1,34 @@
+import { forwardRef } from "react";
```

Comment:
A few things in this example were flagged by `yarn lint` but I made some additional useful improvements that I think will be helpful for most users who will copy/paste this example.